components/templates/Info.tsx

```diff
@@ -0,0 +1,16 @@
+export const Info = ({
+  genesis
+}: {
+  genesis: any
+}) => {
+  return (
+    <div className="flex w-full justify-center text-white">
+        <div className="ccontent-center justify-items-start px-1 py-6 sm:px-12">
+            <div
+                className="genesis"
+                dangerouslySetInnerHTML={{ __html: genesis.body.html }}
```

Comment:
Could we switch this to using `{children}`? That way the page has more control over the informational content (including the ability to add markup for tool tips.

Reply:
Note that without #80, the parent component will still need to render the lesson content like this:
```js
<Info>
	<div dangerouslySetInnerHTML={{ __html: genesis.body.html }}/>
</Info>
```